lib/deep-equal.test.js

```diff
@@ -830,4 +812,76 @@ describe("deepEqual", function() {
             assert.isFalse(checkDeep);
         });
     });
+
+    describe("arrays", function() {
+        function arrayOf(Klass) {
+            return function(value) {
+                return new Klass(value);
+            };
+        }
+        forEach(ARRAY_TYPES, function(value, key) {
```

Comment:
The second argument to [`Array.prototype.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) is index.

```suggestion
        forEach(ARRAY_TYPES, function(value, index) {
```